src/store/actions.js

```diff
@@ -0,0 +1,4 @@
+export const POST_TODO = 'POST_TODO';
```

Comment:
"Action" in the Redux terminology is an object, typically with "type" and optionally "payload".
In this file each constant is an actionType.